docs/metrics/llm-prompt-based-metric.md

```diff
@@ -0,0 +1,174 @@
+# LLM Prompt-based Metric
+
+Prompting Engineering is a technique that involves building and optimizing specific input prompts to allow our Large
+Language Model (LLM) to perform better at targeted tasks. In the landscape of LLM evaluations, prompt engineering is
+designed to envoke the LLMs inherently knowledge and ability to reason to determine whether or not a set of inputs
+contain some sort of hallucination.
+
+## LLM Configurations
```

Comment:
Are these configurations important? I feel like it's less relevant to the metric itself.
If they are relevant and important, in each implementation section, we should explain why we chose certain value for each parameter.

Reply:
I'll remove them since they are not directly relevant to evaluating hallucinations but just general LLM knowledge.